contract_payment_mode/models/contract.py

```diff
@@ -15,28 +15,11 @@ class ContractContract(models.Model):
     def on_change_partner_id(self):
         self.payment_mode_id = self.partner_id.customer_payment_mode_id.id
 
-    @api.multi
     def _prepare_invoice(self, date_invoice, journal=None):
-        invoice_vals = super(ContractContract, self)._prepare_invoice(
+        res = super(ContractContract, self)._prepare_invoice(
```

Comment:
This is not necessary in python3 when using the same class and self field
```suggestion
        res = super()._prepare_invoice(
```

Reply:
Please unfold directly the returned variables:
```suggestion
        invoice_vals, move_form = super()._prepare_invoice(
```
and then do:
```python
return invoice_vals, move_form
```

Reply:
Ok, I have applied these changes :+1: